package.json

```diff
@@ -31,6 +31,7 @@
   "homepage": "http://material-ui.com/",
   "dependencies": {
     "inline-style-prefixer": "^0.6.6",
+    "lodash.flowright": "^3.2.1",
```

Comment:
@oliviertassinari lodash only bumps it's tiny packages if they actually suffer a change in a way. that means even if lodash goes to v5 it may still remain like this and then sudenly bump to v6. I think its very clever and intuitive of them to go through such pain to implement that :+1: